google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/BigtableInstanceAdminClientTests.java

```diff
@@ -81,7 +81,7 @@
 import org.mockito.stubbing.Answer;
 
 @RunWith(JUnit4.class)
-public class BigtableInstanceAdminClientTest {
+public class BigtableInstanceAdminClientTests {
```

Comment:
Why does this need to be renamed? Same for the other pluralizations

Reply:
Graalvm at the moment is incompatible with EasyMock and Mockito. We had to rename this class to exclude it from native image testing. As mentioned above, we use this naming convention to enable native image testing for tests: `IT*` and `*ClientTest`

Reply:
As above, this needs to be documented and all of the classes that dont aren't compatible need to have an explicit marker that they are not run by graalvm. Pluralizing the classname is too subtle

Reply:
Thanks for the suggestion @igorbernstein2! I agree that the class name change is a little too subtle. I have added a javadoc detailing the exclusion.

Reply:
@Neenu1995 Didn't we have a guide about naming convention for integration tests for client libraries?